frontend/index.d.ts

```diff
@@ -47,6 +54,28 @@ interface CAPIType {
             userMessage: string;
         }
     },
+    tags: Array<TagType>
+}
+
+interface TagPropertiesType {
+    id: string,
+    url: string,
+    tagType: string,
+    sectionId: string,
+    sectionName: string,
+    webTitle: string,
+    webUrl: string,
+    twitterHandle?: string,
+    bio?: string,
+    description?: string,
+    emailAddress?: string,
+    contributorLargeImagePath?: string,
+    bylineImageUrl?: string,
+    paidContentType?: string
+}
+
+interface TagType {
```

Comment:
Looks like we're going to have a conflict of sorts here as I've also defined a new `TagType` in my branch https://github.com/guardian/dotcom-rendering/pull/192/files#diff-978893e81256a8dbee76e6fe1045f2d7R28. I'm happy to hang on merging mine until this is in and I'll use this new Type, as my `TagType` is pretty basic.

Reply:
if that's ok then great! happy to wait for you as well though